recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
 
 package:
   name: usgscsm
-  version: "0.1.0"
```

Comment:
Is there a reason why we are replacing the version number with the commit hash too? I'd image we want new_version.commit_hash for our dev builds. So right now, our dev builds would be 1.1.0.#######

Reply:
The GIT_DESCRIBE_TAG should be '1.0.0' based on the 12.19.18 release. Then we should be seeing this getting pushed as `1.0.0-<GIT_DESCRIBE_HASH>-0` as far as I understand.  Our last tag is 1.0.0 and we are overriding the base conda string with the git env variable. The alternative would be to use the `tag-<default_conda_has>-<GIT_DESCRIBE_NUMBER>` to count the number of commits since the last tag. Both will be unique for the dev branch though the former probably provides better info if we are debugging an issue since we will have the short hard.

Reply:
I think this is acceptable then.